### PR TITLE
update docs for istioctl proxy-config cmd 

### PR DESCRIPTION
**Pull request by DreamRivulet**
@liamawhite , could you please help to take a review ?

### CODE REVIEW

File: content/help/ops/component-debugging/index.md [hunk 1]
```diff
@@ -10,24 +10,49 @@ how to get under the hood.
...
+You can get an overview of your mesh via `proxy-status` command:
 
 {{< text bash >}}
-$ istioctl proxy-config endpoint <pod-name> clusters
+$ istioctl proxy-status
 {{< /text >}}
 
-To retrieve endpoints for a given pod in the application namespace from Pilot, run the following command:
+If a proxy is missing from the output list it means that it is not currently connected to a Pilot instance so will not be receiving any configuration.
+Also if it is stale that likely means there are networking issues or Pilot needs to be scaled.
```
Comment (liamawhite) [inline content/help/ops/component-debugging/index.md]: Maybe keep this as a single sentence.

Comment (DreamRivulet) [reply content/help/ops/component-debugging/index.md]: ok, let me change.

File: content/help/ops/component-debugging/index.md [hunk 2]
```diff
@@ -10,24 +10,49 @@ how to get under the hood.
...
+$ istioctl proxy-config route <pod-name> [flags]
 {{< /text >}}
 
-The `proxy-config` command also allows you to retrieve the state of the entire mesh from Pilot using the following command:
+To retrieve information about endpoint configuration for the Envoy instance in the specified pod, run the following command:
 
 {{< text bash >}}
-$ istioctl proxy-config pilot mesh ads
+$ istioctl proxy-config endpoints <pod-name> [flags]
 {{< /text >}}
```
Comment (liamawhite) [inline content/help/ops/component-debugging/index.md]: More information about how to interpret this information can be found [here](/help/ops/traffic-management/proxy-cmd/)

I think this is the correct way to link. Please correct if I'm wrong @geeknoid.

Comment (DreamRivulet) [reply content/help/ops/component-debugging/index.md]: shoudn't  the link be /help/ops/traffic-management/proxy-cmd/ ?

Comment (frankbu) [reply content/help/ops/component-debugging/index.md]: https://istio.io/help/ops/traffic-management/proxy-cmd/

File: content/help/ops/component-debugging/index.md [hunk 3]
```diff
@@ -10,26 +10,52 @@ how to get under the hood.
 
 ## With `istioctl`
 
-`istioctl` allows you to inspect the current xDS of a given Envoy from its admin interface (locally) or from Pilot using the `proxy-config` or `pc` command.
+### Get an overview of your mesh
 
-For example, to retrieve the configured clusters in an Envoy via the admin interface run the following command:
+You can get an overview of your mesh via `proxy-status` command:
```
Comment (geeknoid) [inline content/help/ops/component-debugging/index.md]: your mesh using the `proxy-status` command

File: content/help/ops/component-debugging/index.md [hunk 4]
```diff
@@ -10,26 +10,52 @@ how to get under the hood.
...
 
 {{< text bash >}}
-$ istioctl proxy-config endpoint <pod-name> clusters
+$ istioctl proxy-status
 {{< /text >}}
 
-To retrieve endpoints for a given pod in the application namespace from Pilot, run the following command:
+If a proxy is missing from the output list it means that it is not currently connected to a Pilot instance so will not be receiving any configuration, also if it is stale that likely means there are networking issues or Pilot needs to be scaled.
```
Comment (geeknoid) [inline content/help/ops/component-debugging/index.md]: so will not -> so it will not

File: content/help/ops/component-debugging/index.md [hunk 5]
```diff
@@ -10,26 +10,52 @@ how to get under the hood.
...
+
+### Get proxy configuration  
+
+`istioctl` allows you to retrieve information about proxy configuration from the Envoy config dump using the `proxy-config` or `pc` command.
+
+For example, to retrieve information about cluster configuration for the Envoy instance in the specified pod, run the following command:
```
Comment (geeknoid) [inline content/help/ops/component-debugging/index.md]: in the specified pod -> in a specific pod

File: content/help/ops/component-debugging/index.md [hunk 6]
```diff
@@ -10,26 +10,52 @@ how to get under the hood.
...
+
+{{< text bash >}}
+$ istioctl proxy-config cluster <pod-name> [flags]
+{{< /text >}}
+
+To retrieve information about bootstrap configuration for the Envoy instance in the specified pod, run the following command:
```
Comment (geeknoid) [inline content/help/ops/component-debugging/index.md]: in the specified -> in a specific

File: content/help/ops/component-debugging/index.md [hunk 7]
```diff
@@ -10,26 +10,52 @@ how to get under the hood.
...
+
+{{< text bash >}}
+$ istioctl proxy-config bootstrap <pod-name> [flags]
+{{< /text >}}
+
+To retrieve information about listener configuration for the Envoy instance in the specified pod, run the following command:
```
Comment (geeknoid) [inline content/help/ops/component-debugging/index.md]: in the specified -> in a specific

File: content/help/ops/component-debugging/index.md [hunk 8]
```diff
@@ -10,26 +10,52 @@ how to get under the hood.
...
+
+{{< text bash >}}
+$ istioctl proxy-config listener <pod-name> [flags]
+{{< /text >}}
+
+To retrieve information about route configuration for the Envoy instance in the specified pod, run the following command:
```
Comment (geeknoid) [inline content/help/ops/component-debugging/index.md]: the specified -> a specific

File: content/help/ops/component-debugging/index.md [hunk 9]
```diff
@@ -10,26 +10,52 @@ how to get under the hood.
...
 
 {{< text bash >}}
-$ istioctl proxy-config pilot -n application <pod-name> eds
+$ istioctl proxy-config route <pod-name> [flags]
 {{< /text >}}
 
-The `proxy-config` command also allows you to retrieve the state of the entire mesh from Pilot using the following command:
+To retrieve information about endpoint configuration for the Envoy instance in the specified pod, run the following command:
```
Comment (geeknoid) [inline content/help/ops/component-debugging/index.md]: the specified -> a specific

File: content/help/ops/component-debugging/index.md [hunk 10]
```diff
@@ -10,26 +10,52 @@ how to get under the hood.
...
 
 {{< text bash >}}
-$ istioctl proxy-config pilot mesh ads
+$ istioctl proxy-config endpoints <pod-name> [flags]
 {{< /text >}}
 
+More information about how to interpret this information can be found [here](/help/ops/traffic-management/proxy-cmd/).
```
Comment (geeknoid) [inline content/help/ops/component-debugging/index.md]: In general, [it's better not to use "here" as a link name](http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/). In that context, I suggest something like:

See [page title](page url) for more advice on interpreting this information.

Comment (DreamRivulet) [reply content/help/ops/component-debugging/index.md]: Thanks for your comments, i submitted the new change, could you please help to review?

**Pull request merged**